arrow-data-source/script/build_arrow.sh

```diff
@@ -62,7 +62,7 @@ echo "ARROW_SOURCE_DIR=${ARROW_SOURCE_DIR}"
 echo "ARROW_INSTALL_DIR=${ARROW_INSTALL_DIR}"
 mkdir -p $ARROW_SOURCE_DIR
 mkdir -p $ARROW_INSTALL_DIR
-git clone https://github.com/oap-project/arrow.git  --branch arrow-4.0.0-oap $ARROW_SOURCE_DIR
+git clone https://github.com/philo-he/arrow.git  --branch floor $ARROW_SOURCE_DIR
```

Comment:
remove this?

Reply:
Just reverted it at latest commit. Thanks!